config/locales/server.be.yml

```diff
@@ -1239,7 +1239,7 @@ be:
     staff_user_custom_fields: "Спіс карыстальнікаў карыстацкіх палёў, якія могуць быць атрыманы для супрацоўнікаў з API."
     enable_user_directory: "Пакажыце каталог карыстальнікаў для прагляду"
     enable_group_directory: "Забяспечыць каталог груп для прагляду"
-    enable_category_group_review: "Дазволіць групы для разгляду ўтрымання ў канкрэтных катэгорыях"
```

Comment:
We needn't check in these things for other languages - they will be automatically flagged in transifex when the keys change.